CONTRIBUTING.rst

```diff
@@ -23,35 +23,64 @@ Policy objectives
+
+4. Contributions must be reviewed by every willing organizations
+   (Default reviewers are `@tlvu`_ for `Ouranos`_ and `@MatProv`_ for `CRIM`_).
 
 .. _@tlvu: https://github.com/tlvu
 .. _Ouranos: https://github.com/Ouranosinc
 .. _@MatProv: https://github.com/MatProv
 .. _CRIM: https://github.com/crim-ca
 
 5. The reviews must be rigorous while respecting the initial scope.
-6. Each organization wishing to review the changes has the duty to do so within a reasonable period of time (7 days) or to indicate its intention to do so later with reasonable reasons (e.g., vacation). After this time, its implicit support will be considered. It will be assumed that the organization agrees to the changes, and they will get merged without further notice.
-7. To encourage review in a timely manner, contributions should be simple and focused (do not mix multiple goals) and well explained (describe the "why" and the "impact/behavior change" on existing production deployment, as requested in the contribution template).
-8. Each organization maintains a fork for its production allowing it to deploy the platform at its own pace. It also allows to self-manage the production fork contribution permissions and develop feature branches.
-9. Each organization is responsible for keeping its production fork up to date with the main branch to avoid discrepancies.
-10. If patches or contributions are made directly in the production fork, they must also be ported back and approved in the main branch (no code that does not exist in the main branch should exist in a production fork).
-11. The main branch will contain the official versions of PAVICS that will evolve according to semantic versioning. These versions should be used by the organizations.
-12. If contributions are made directly in a production fork (point 10), a tagged version should use the last common one with the main branch but also include a suffix.
-    * Example: The main branch is at `2.1.8`, and a contribution is made in a production fork from `2.1.8`. The tag `2.1.9` cannot be applied because this version could possibly exists in the main branch. A tag looking like `2.1.8.orgXrev1` would be preferred.
+
+6. Each organization wishing to review the changes has the duty to do so within a reasonable period of time (7 days)
+   or to indicate its intention to do so later with reasonable reasons (e.g., vacation). After this time, its implicit
+   support will be considered. It will be assumed that the organization agrees to the changes, and they will get merged
+   without further notice.
+
+7. To encourage review in a timely manner, contributions should be simple and focused (do not mix multiple goals) and
+   well explained (describe the "why" and the "impact/behavior change" on existing production deployment, as requested
+   in the contribution template).
+   They should also include a description of provided modifications and fixes under the active ``Unreleased`` section
```

Comment:
If possible, can you perform "reformating" in a separate commit so it's easier to spot what has changed by the reformatting and what has really changed by manual update.